.github/workflows/pull_request.yaml

```diff
@@ -99,3 +99,31 @@ jobs:
           PY_COLORS: '1'
           ANSIBLE_FORCE_COLOR: '1'
         shell: bash
+  vagrant:
```

Comment:
I'm wondering if we should run for branches instead of PRs

Reply:
So we'd test immediately after merging?

I'm open to that. I figured we'd adjust the schedule according to dev feedback. Or look into enabling merging while these tests still have not completed yet.

Reply:
sounds good to me